addons/bus/controllers/main.py

```diff
@@ -41,9 +41,9 @@ def poll(self, channels, last, options=None):
             raise exceptions.UserError(_("bus.Bus not available in test mode"))
         return self._poll(request.db, channels, last, options)
 
-    @route('/longpolling/im_status', type="json", auth="user")
+    @route('/longpolling/im_status', type="json", auth="public")
```

Comment:
if you are going to make it public, should add more verifications, not add a sudo